services/brig/src/Brig/API/Error.hs

```diff
@@ -366,8 +366,9 @@ authTokenUnsupported = Wai.mkError status403 "invalid-credentials" "Unsupported
 insufficientTeamPermissions :: Wai.Error
 insufficientTeamPermissions = errorToWai @'E.InsufficientTeamPermissions
 
-noBindingTeam :: Wai.Error
-noBindingTeam = Wai.mkError status403 "no-binding-team" "Operation allowed only on binding teams"
+-- FUTUREWORK: should this status be 404?
+teamNotFound :: Wai.Error
+teamNotFound = Wai.mkError status403 "no-binding-team" "Operation allowed only on binding teams"
```

Comment:
Your FUTUREWORK is correct:

```suggestion
teamNotFound :: Wai.Error
teamNotFound = notFound "Team not found."
```

You probably have to also fix some `CanThrow`s if you do this, right?

Reply:
It's only used in the internal POST /i/legalhold-login` route. So there is no swagger.